### ISSUE TITLE
Weekly Scrum - Mar 6 - 10

### ISSUE THREAD

**Issue by kfoxIsProgrammer** [SYSC4806-Group13/SYSC4806-Project#44]
Each week a group member should open a new issue called Weekly scrum – [date], then every team
member should add comments with their own contribution, i.e. their answers to the questions:

1. what have I done this week? (with link(s) to the relevant GitHub Issue(s))

2. what will I do next week? (with link(s) to the relevant GitHub Issue(s))

3. what is holding me back? (if applicable)

Expected length: 1-2 sentences per item

**Comment by Justgo13** [comment]
what have I done this week? (with link(s) to the relevant GitHub Issue(s))
Completed the frontend portion for this story #49 
* Reviewed #10 and #9 
* Helped with ER diagrams
* Helped with UML diagrams

what will I do next week? (with link(s) to the relevant GitHub Issue(s))
* Create the jacard distance for suggesing books to users based on other users

what is holding me back? (if applicable)
Blocked on waiting for checkout system and user table in DB

**Comment by MillanWang** [comment]
- **What have I done this week? (with link(s) to the relevant GitHub Issue(s))**
  - Merged in PR for linking CartItems and Listings with updates to make reusable controller utilities #37 
  - Created UML diagrams 
  - Created ER diagrams
  - Updated README to summarize the plans for future milestones 
  - PR Reviews
  - Updated Kanban board to divide labor in a feature based approach instead of a technical level based approach.
- **What will I do next week? (with link(s) to the relevant GitHub Issue(s))**
  - Complete checkout system ticket #15 
  - Integration with the OAuth and user management system
  - Likely some PR reviews
- **What is holding me back? (if applicable)**
  - Getting the OAuth PRs into the main branch. The checkout system depends on this but it's a huge change that completely reworks the utilization of the app

**Comment by kfoxIsProgrammer** [comment]
1. what have I done this week? (with link(s) to the relevant GitHub Issue(s))
- I had updated my PRs for https://github.com/SYSC4806-Group13/SYSC4806-Project/pull/36 and https://github.com/SYSC4806-Group13/SYSC4806-Project/pull/28. 
- I reviewed Jason's https://github.com/SYSC4806-Group13/SYSC4806-Project/pull/35 PR.

2. what will I do next week? (with link(s) to the relevant GitHub Issue(s))
- Waiting to merge OAUTH
- Dealing with Frontend updates for Logging in

4. what is holding me back? (if applicable)
- No blockers.

**Comment by NosBos** [comment]
- **What have I done this week**
  -   Worked on #38 and am mostly done. Just doing some validation to ensure only valid image types are accepted.
  -   Reviewed PRs: #9 #10 
- **What will I do next week**
  -   Complete PR for #38 
- **Blockers**
  -   No

**Comment by GolanHassin** [comment]
**What have I done this week? (with link(s) to the relevant GitHub Issue(s))**

- Began work on #47. It is a work in progress and will be done by the end of the next week.
- Also began planning and some work for #48 
- Testing of other branches
- Began writing tests for both branches

- **What will I do next week? (with link(s) to the relevant GitHub Issue(s))**
  - I will complete the PR for #47  
  - I will complete the PR for #48  
  - I will add tests for #47 
  - I will add tests for #48
- **What is holding me back? (if applicable)**
  - Performing merges with the main integration branches when a PR is accepted
